src/content/docs/tutorials/crates-tui/app_handle_event.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Handle Events
+---
+
+In the `run` method, we have the following code:
+
+```rust
+if let Some(e) = events.next().await {
+    self.handle_event(e)?.map(|action| self.tx.send(action));
+}
+```
+
+`events.next().await` returns an `Event` from the stream we implemented in the previous section.
+This `event` is passed to the `handle_event` method.
+
+`handle_event` returns an `Option<Action>` which we map to an `Action` that has to be acted on by
```

Comment:
What about calling this translate_event? Handle implies that some action should happen as a result of the event, but these methods are purely about translating the event into an action that will be executed.

Reply:
Good question. I think prefer `handle_*` since it's easier to understand in the context of this tutorial, other tutorials and code that a user is going to write.